### ISSUE TITLE
Puppeteer fails to install and run on ARM64 chips

### ISSUE THREAD

**Issue by almchung** [processing/p5.js#5409]
#### Most appropriate sub-area of p5.js?

- [ ] Accessibility (Web Accessibility)
- [x] Build tools and processes
- [ ] Color
- [ ] Core/Environment/Rendering
- [ ] Data
- [ ] DOM
- [ ] Events
- [ ] Friendly error system
- [ ] Image
- [ ] IO (Input/Output)
- [ ] Localization
- [ ] Math
- [x] Unit Testing
- [ ] Typography
- [ ] Utilities
- [ ] WebGL
- [ ] Other (specify if possible)

#### Details about the bug:
`puppeteer` package (v^5.5.0) would fail to install and run on ARM64 chip machines. I encountered this error when running our headless unit tests using `npm run test` commend, as recommended by our official documents. 

This error has been found on machines with the new Apple Silicon/M1 and Raspberry Pi4. The cause seems to be missing ARM builds for Chromium, and it is unclear the build will be available soon. On a local level, some lengthy remedy has been suggested ([LINK](https://github.com/puppeteer/puppeteer/issues/6622)) but wouldn't work for everyone. In the same thread, there is a report that updating the dependency of `puppeteer` to v^10.0.0 would resolve the issue ([LINK](https://github.com/puppeteer/puppeteer/issues/6622#issuecomment-906476717)).

On a separate but related note, I will file another issue to gauge interest in adding the Mocha+Playwright option to the current Mocha+Puppeteer.

- p5.js version: v1.4.0
- Web browser and version: Chromium
- Operating System: MacOSX 11.4
- Steps to reproduce this:
1. On a ARM64 chip machine, clone the latest version of p5.js.
2. Install node (v16.6.1) and npm (v7.20.5)
3. `npm install grunt`
4. `npm run test` ⇝ Fatal Error! (see below)
```Running "mochaChrome:yui" (mochaChrome) task
Fatal error: Failed to launch the browser process! spawn /usr/bin/chromium-browser ENOENT


TROUBLESHOOTING: https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md

**Comment by limzykenneth** [comment]
I do plan on updating puppeteer to the latest version (the version we are using doesn't work with node 16) but got some failed tests that stopped me from proceeding. Details in #5367. If those tests can be solved, we can update to the latest version of puppeteer.

**Comment by almchung** [comment]
@limzykenneth I tried to fix the unit tests issue here: #5412. Hopefully, this remedies some of them!

**Comment by limzykenneth** [comment]
I'll update puppeteer to 10.2.0 once I've checked a few things and update other outdated packages as well, which should solve this issue.
